aten/src/ATen/native/native_functions.yaml

```diff
@@ -4512,6 +4512,13 @@
     CompositeExplicitAutogradNonFunctional: select_backward
   autogen: select_backward.out
 
+- func: _nested_select_backward(Tensor grad_output, Tensor self, int dim, int index) -> Tensor
+  variants: function
+  device_check: NoCheck
+  device_guard: False
+  dispatch:
+    NestedTensorCPU, NestedTensorCUDA: _nested_select_backward
```

Comment:
does this mean a NestedTensor key rather than a per-device NestedTensor{device} keys?

Reply:
yep! It seems pretty common to write a kernel that works for all nested tensor backends, so it would be nice to be able to write it in the yaml as `NestedTensor: kernel`